guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -66,13 +66,15 @@ The release notes include:
 
 -   {:.fix}**Environment variable updates**—Changed the following environment variables:
 
-    -  {:.new}<!-- MAGECLOUD-3026 & MAGECLOUD-2963-->Added the [RESOURCE_CONFIGURATION environment variable]({{page.baseurl}}/cloud/env/variables-deploy.html#resource_configuration) to map a resource name to a database connection.
+    -  {:.new}<!-- MAGECLOUD-3026 & MAGECLOUD-2963-->**[RESOURCE_CONFIGURATION deploy environment variable]({{page.baseurl}}/cloud/env/variables-deploy.html#resource_configuration)**—New variable to map a resource name to a database connection. 
```

Comment:
Since we have multiple NEW ones and we want to draw attention to them separately, you could create a separate section. That way we don't have to have the icons in the sub-bullets. That was my error on an earlier commit. I did that because there was only one new one and I still wanted people to know it was new, but it actually doesn't look that great.